xbmc/addons/PVRClient.h

```diff
@@ -217,6 +217,33 @@ namespace PVR
      */
     PVR_ERROR RenameChannel(const CPVRChannelPtr &channel);
 
+    /*
+     * @brief Check if an epg tag can be recorded
+     * @param tag The epg tag
+     * @param isRecordable Set to true if the tag can be recorded
+     * @return PVR_ERROR_NO_ERROR if isRecordable was set by the addon
+     */
+    PVR_ERROR IsRecordable(const EPG::CConstEpgInfoTagPtr &tag, bool *isRecordable);
```

Comment:
Ooops, just realized that we are in CPVRClient here. Sooo it's quite opposite. Use bool& instead of bool*, const ref tag is okay.